apps/wallet/package.json

```diff
@@ -96,6 +97,7 @@
         "eslint-plugin-react": "^7.31.8",
         "formik": "^2.2.9",
         "framer-motion": "^7.5.1",
+        "lodash-es": "^4.17.21",
```

Comment:
Don't love including lodash for the sake of debouncing. It's pretty easy to implement debounce without it.

Reply:
I agree, but I assume it will only include the debouncing functionality so shouldn't too bad. You don't like lodash or don't want to include a library for this?

Reply:
Yeah, it's not really enough of a concern to block the PR, it should tree-shake everything else out which is good.

For utility functions, I generally just prefer to implement them internally, or pull in purpose-built packages.

For collections, I try to avoid Lodash. It's not well maintained anymore, doesn't have native TS support, has had breaking changes released under patch versions, etc.

Reply:
I will see if there is any other library and use that one - otherwise do it myself